### PR TITLE
how-to/deploy: add arm for deployment by tidb-ansible

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)

Add ARM for deployment by TiDB Ansible.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v4.0 changes for now)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

**If you select two or more versions from above**, to trigger the bot to cherry-pick this PR to your desired release version branch(es), you **must** add corresponding labels such as **needs-cherry-pick-3.1**, **needs-cherry-pick-3.0**, and **needs-cherry-pick-2.1**.

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/2501  https://github.com/pingcap/docs-cn/pull/2515
- Other reference link(s):

### CODE REVIEW

File: how-to/deploy/orchestrated/ansible.md [hunk 1]
```diff
@@ -616,8 +616,9 @@ To enable the following control variables, use the capitalized `True`. To disabl
 | Variable Name | Description |
 | :---- | :------- |
 | cluster_name | the name of a cluster, adjustable |
+| cpu_architecture | CPU architecture. `amd64` by default, `arm64` optional. |
```
Comment (ran-huang) [inline how-to/deploy/orchestrated/ansible.md:619]:
```suggestion
| cpu_architecture | CPU architecture. `amd64` by default, `arm64` optional |
```

File: how-to/deploy/orchestrated/ansible.md [hunk 2]
```diff
@@ -616,8 +616,9 @@ To enable the following control variables, use the capitalized `True`. To disabl
 | Variable Name | Description |
 | :---- | :------- |
 | cluster_name | the name of a cluster, adjustable |
```
Comment (ran-huang) [inline how-to/deploy/orchestrated/ansible.md:618]: How about enclosing the variables with ``` ` ```?
```suggestion
| `cluster_name` | the name of a cluster, adjustable |
```

Comment (TomShawn) [reply how-to/deploy/orchestrated/ansible.md:618]: Strictly, parameter names should be enclosed with ``. But none of the parameters in this document table are so enclosed.

Comment (ran-huang) [reply how-to/deploy/orchestrated/ansible.md:618]: OK

**Pull request merged**